ophyd/v2/core.py

```diff
@@ -75,6 +77,10 @@ def _run_callbacks(self, task: asyncio.Task):
             for callback in self._callbacks:
                 callback(self)
 
+    @property
+    def exception(self) -> Optional[StatusException]:
```

Comment:
Note to self, this matches https://docs.python.org/3/library/concurrent.futures.html#concurrent.futures.Future.exception